src/OSPSuite.R/Services/ContainerTask.cs

```diff
@@ -18,6 +18,7 @@ public interface IContainerTask
       IContainer[] AllContainersMatching(IModelCoreSimulation simulation, string path);
       IQuantity[] AllQuantitiesMatching(IModelCoreSimulation simulation, string path);
       IMoleculeAmount[] AllMoleculesMatching(IModelCoreSimulation simulation, string path);
+      IParameter[] AllSensitivityAnalysisParametersMatching(ISimulation simulation, string path);
```

Comment:
I have added this one here that returns all SA parameters with a  wild card
Not useful for reporting engine but probably neat when checking only the influence of parameters withing a specifc organ for example

Reply:
Will this return ALL parameters matching the path, or only meaningful (used in model, non-categorical etc.)?

Reply:
yes. I am going to rename it to AllParametersForSensitivityAnalysisMatching. This makes more sense I believe

Reply:
>Will this return ALL parameters matching the pat 

No that would not make sense. This is AllParametersMatching